doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -1168,6 +1168,7 @@ The `mz_webhook_sources` table contains a row for each webhook source in the sys
 [`timestamp with time zone`]: /sql/types/timestamp
 
 <!-- RELATION_SPEC_UNDOCUMENTED mz_internal.mz_activity_log_thinned -->
+<!-- RELATION_SPEC_UNDOCUMENTED mz_internal.mz_cluster_replica_metrics_history_ct -->
```

Comment:
I think there's a lint that enforces this stays updated, if so we should add Continual Tasks to it

Reply:
Is it this one? If so, looks it works based on something hooked up in this PR https://buildkite.com/materialize/test/builds/92813#0192a101-493d-4594-bd72-b44116bbe164

Reply:
Exactly!